src/platforms/flutter/migration.mdx

```diff
@@ -3,6 +3,10 @@ title: Migration Guide
 sidebar_order: 1000
 ---
 
+## Migrating From `sentry_flutter` `6.12.x` to `sentry` `6.13.x`
+
+Errors that are not caught by Flutter are recorded using [`PlatFormdispatcher.onError`](https://docs.flutter.dev/testing/errors#errors-not-caught-by-flutter). The SDK will install it's own callback and record errors automatically for Flutter versions 3.3 and above. Please consider this, if you yourself want to use the callback. For Flutter versions lower than 3.3, the `AppRunner` you provide on init will be run inside a custom zone using [`runZonedGuarded`](https://api.flutter.dev/flutter/dart-async/runZonedGuarded.html) to records errors.
```

Comment:
I'd rather explain that on Flutter < 3.3 we keep using the runZonedGuarded and after that the new onError, also that no changes are needed.